src/main/java/core/basesyntax/service/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        String fruit = transaction.getFruit();
+        int amount = transaction.getAmount();
+
+        storage.put(fruit, storage.getOrDefault(fruit, 0) + amount);
```

Comment:
You are directly manipulating the storage in the `handle` method. This violates the Single Responsibility Principle. The `handle` method should only be responsible for handling operations, not manipulating storage. Consider passing the storage manipulation responsibility to another method or class.